api/migrations/0007_lessonstudent_mood_analyzer.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.1.1 on 2020-09-13 19:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0006_remove_student_age'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='lessonstudent',
+            name='mood_analyzer',
+            field=models.CharField(default='', max_length=300),
```

Comment:
I had to drop the db since I originally had the default as null and it did not update so I changed it to an empty string and then it worked